risc0/bigint2/src/lib.rs

```diff
@@ -17,6 +17,8 @@ pub mod ec;
 #[cfg(feature = "unstable")]
 pub mod ffi;
 #[cfg(feature = "unstable")]
+pub mod field;
```

Comment:
Thinking out loud that it might be that this crate is getting larger than needed, where it might be ideal before removing `unstable` feature flag, to have these be separated into their own feature flags or separate crates. Nothing to do on this though, and opinionated